libraries/streaming-e2e-tests/bot/utils/generateDirectLineToken.js

```diff
@@ -0,0 +1,47 @@
+const createUserId = require('./createUserId');
+
+module.exports = async function (
+  directLineSecret = process.env.DIRECT_LINE_SECRET,
+  { domain = process.env.DIRECT_LINE_URL || 'https://directline.botframework.com/', userId = createUserId() } = {}
```

Comment:
Should this function always generate a DL ASE token?

```js
domain = `https://${process.env.WEBSITE_HOSTNAME}.azurewebsites.net/.bot/`
```

Reply:
Hmm yeah. Considering we want to test DL ASE here 😅 
The copy pasta strikes again!

Changing it to the DL ASE token gen endpoint. Thanks William